extensions/answer_summarizers/models_test.py

```diff
@@ -16,15 +16,27 @@
 
 """Test calculations to get interaction answer views."""
 
+import itertools
+
+from extensions.answer_summarizers import models as answer_models
```

Comment:
This should go below "from core.tests ..." (Alphabetize)

Reply:
Done.